### DISCUSSION TITLE
Constant current LED driver

### DISCUSSION THREAD

**Discussion by tedlanghorst** [tedlanghorst/OpenOBS#21] [Ideas]
The voltage drop of the IRED changes over time and with temperature. Including a constant current driver should help the temperature dependence and reduce drift over long deploys. [LM134](https://www.ti.com/lit/ds/symlink/lm334.pdf?ts=1646844788906&ref_url=https%253A%252F%252Fwww.ti.com%252Fproduct%252FLM334%253Futm_source%253Dgoogle%2526utm_medium%253Dcpc%2526utm_campaign%253Dapp-null-null-GPN_EN-cpc-pf-google-wwe%2526utm_content%253DLM334%2526ds_k%253DLM334%2526DCM%253Dyes%2526gclid%253DEAIaIQobChMI8bqS57659gIV8pFbCh2N-gS3EAAYASAAEgJ0_PD_BwE%2526gclsrc%253Daw.ds) might be a good option

**Comment by ebgoldstein** [comment]
is the package the same size? or would it require changing the PCB?
